kuksa_go_client/main.go

```diff
@@ -22,105 +22,101 @@ import (
 	"log"
 
 	"github.com/eclipse/kuksa.val/kuksa_go_client/kuksa_client"
-	"github.com/eclipse/kuksa.val/kuksa_go_client/protocInstall"
 )
 
 func main() {
+log.Println("Starting Kuksa Client!!")
+
+// Load the Kuksa Configuration
+var configKuksaClient kuksa_client.KuksaClientConfig
+kuksa_client.ReadConfig(&configKuksaClient)
+protocol := flag.String("protocol", "undefined", "Could be grpc or ws. Per default the protocol variable is undefined. Then the value from the config file kuksa_client.json will be used.")
+flag.Parse()
+if *protocol == "undefined" {
+	protocol = &configKuksaClient.TransportProtocol
+}
+
+var backend kuksa_client.KuksaBackend
+
+var token string
+if *protocol == "ws" {
+	// example token from kuksa.val/kuksa_certificates/jwt/all-read-write.json.token
+	token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJrdWtzYS52YWwiLCJpc3MiOiJFY2xpcHNlIEtVS1NBIERldiIsImFkbWluIjp0cnVlLCJpYXQiOjE1MTYyMzkwMjIsImV4cCI6MTc2NzIyNTU5OSwia3Vrc2EtdnNzIjp7IioiOiJydyJ9fQ.QQcVR0RuRJIoasPXYsMGZhdvhLjUalk4GcRaxhh3-0_j3CtVSZ0lTbv_Z3As5BfIYzaMlwUzFGvCVOq2MXVjRK81XOAZ6wIsyKOxva16zjbZryr2V_m3yZ4twI3CPEzJch11_qnhInirHltej-tGg6ySfLaTYeAkw4xYGwENMBBhN5t9odANpScZP_xx5bNfwdW1so6FkV1WhpKlCywoxk_vYZxo187d89bbiu-xOZUa5D-ycFkd1-1rjPXLGE_g5bc4jcQBvNBc-5FDbvt4aJlTQqjpdeppxhxn_gjkPGIAacYDI7szOLC-WYajTStbksUju1iQCyli11kPx0E66me_ZVwOX07f1lRF6D2brWm1LcMAHM3bQUK0LuyVwWPxld64uSAEsvSKsRyJERc7nZUgLf7COnUrrkxgIUNjukbdT2JVN_I-3l3b4YXg6JVD7Y5g0QYBKgXEFpZrDbBVhzo7PXPAhJD6-c3DcUQyRZExbrnFV56RwWuExphw8lYnbMvxPWImiVmB9nRVgFKD0TYaw1sidPSSlZt8Uw34VZzHWIZQAQY0BMjR33fefg42XQ1YzIwPmDx4GYXLl7HNIIVbsRsibKaJnf49mz2qnLC1K272zXSPljO11Ke1MNnsnKyUH7mcwEs9nhTsnMgEOx_TyMLRYo-VEHBDLuEOiBo"
```

Comment:
The hard-coded token here means that we practically ignore what has been defined in https://github.com/eclipse/kuksa.val/blob/master/kuksa_go_client/kuksa-client.json right? As we only use it as backup? Or?

Reply:
Yes, but since these are some examples how to use the go client, I thought this would be the best way to showcase it.